internal/providers/terraform/aws/neptune_cluster.go

```diff
@@ -0,0 +1,94 @@
+package aws
+
+import (
+	"fmt"
+
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/shopspring/decimal"
+	"github.com/tidwall/gjson"
+)
+
+func GetNeptuneClusterRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "aws_neptune_cluster",
+		RFunc: NewNeptuneCluster,
+	}
+}
+
+func NewNeptuneCluster(d *schema.ResourceData, u *schema.UsageData) *schema.Resource {
+	var storageGb, monthlyIoRequests *decimal.Decimal
+	region := d.Get("region").String()
+	costComponents := make([]*schema.CostComponent, 0)
+
+	if u != nil && u.Get("storage_gb").Type != gjson.Null {
+		storageGb = decimalPtr(decimal.NewFromInt(u.Get("storage_gb").Int()))
+	}
+	if u != nil && u.Get("monthly_io_requests").Type != gjson.Null {
+		monthlyIoRequests = decimalPtr(decimal.NewFromInt(u.Get("monthly_io_requests").Int()))
+	}
+
+	costComponents = backupCostComponent(d, u, costComponents, region)
+	costComponents = append(costComponents, neptuneClusterStorageIOsCostComponents("Storage", "GB", region, "StorageUsage", storageGb, 1))
+	costComponents = append(costComponents, neptuneClusterStorageIOsCostComponents("I/O", "1M request", region, "StorageIOUsage", monthlyIoRequests, 1000000))
+
+	return &schema.Resource{
+		Name:           d.Address,
+		CostComponents: costComponents,
+	}
+}
+func backupCostComponent(d *schema.ResourceData, u *schema.UsageData, costComponents []*schema.CostComponent, region string) []*schema.CostComponent {
+	var retentionPeriod *decimal.Decimal
+	if d.Get("backup_retention_period").Exists() || d.References("db_cluster_identifier") != nil {
+		retentionPeriod = decimalPtr(decimal.NewFromInt(d.Get("backup_retention_period").Int()))
+		if retentionPeriod.GreaterThan(decimal.NewFromInt(1)) {
+			var backupStorage *decimal.Decimal
+			if u != nil && u.Get("backup_storage_gb").Type != gjson.Null {
+				backupStorage = decimalPtr(decimal.NewFromInt(u.Get("backup_storage_gb").Int()))
+			}
+			costComponents = append(costComponents, neptuneClusterBackupCostComponent(region, "BackupUsage", backupStorage))
+		}
```

Comment:
I think it would be better if you set this cost component as free resource if `retentionPeriod` will be lower than 1
```suggestion
		}
	else {
		return &schema.Resource{
 		NoPrice:   true,
 		IsSkipped: true,
 	}
```